core/src/main/scala/kafka/utils/CoreUtils.scala

```diff
@@ -277,4 +279,10 @@ object CoreUtils extends Logging {
     val listenerList = parseCsvList(listeners)
     listenerList.map(listener => EndPoint.createEndPoint(listener)).map(ep => ep.protocolType -> ep).toMap
   }
+
+  def wrapInProperties(key: String, value: String) = {
```

Comment:
Maybe this should be `def props(map: Map[String, String])` and then you would use it like:

```scala
CoreUtils.props(Map("foo" -> "blah"))
```